src/components/shared/Widget/Base.vue

```diff
@@ -41,10 +46,54 @@ export default class BaseWidget extends Vue {
   @Prop({ default: false, type: Boolean }) readonly primaryTitle!: boolean;
 
   public capitalize = capitalize;
+  
+  get isPipAvailable() {
+    return 'documentPictureInPicture' in window;
```

Comment:
TODO: check how does it work in Telegram if it's available. If it works bad then we need to add additional check with isTma